### DISCUSSION TITLE
Organization-level Webhook support for Actions

### DISCUSSION THREAD

**Discussion by prototypicalpro** [community/community#26091] [Actions]
<p>Hello! My name is Noah Koontz, and I am currently developing open source management tooling for New Relic using GitHub Actions. I was wondering if there are any plans for supporting <a href="https://docs.github.com/en/rest/reference/orgs#webhooks" rel="nofollow noopener">organization-wide webhooks</a> as triggers for actions? This feature would be particularly useful for events such as <code>public</code> or <code>milestone</code>, which have limited usefulness when scoped to a single repository. Thanks!</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/prototypicalpro">@prototypicalpro</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>Yes, Organization and enterprise workflows are in the roadmap, sorry there is no timeline to share yet.<br>
</p><aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/github/roadmap/issues/52" target="_blank" rel="nofollow noopener">github.com/github/roadmap</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/github/roadmap/issues/52" target="_blank" rel="nofollow noopener">Actions: Organization and enterprise workflows</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2020-07-24" data-time="17:14:29" data-timezone="UTC">05:14PM - 24 Jul 20 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/github-product-roadmap" target="_blank" rel="nofollow noopener">
          <img alt="github-product-roadmap" src="https://user-images.githubusercontent.com/52772761/181095178-7b29a2b7-1898-490c-85f4-b9ebb6b9b83b.png" class="onebox-avatar-inline" width="20" height="20">
          github-product-roadmap
        </a>
      </div>
    </div>
  </div>
</div>

<div class="github-row">
  <p class="github-content">Summary
This feature enables you to define a workflow that is run against multiple repositories
Intended Outcome
This feature is designed to give you...</p>
</div>

<div class="labels">
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">actions</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">cloud</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">code-to-cloud</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">ga</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">github enterprise</span>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thanks</p>

**Comment by jmiller-rise8** [comment]
My org has created https://github.com/marketplace/webhook-actions to allow for dispatching across repositories. Currently we can only offer a free plan until we reach 100 installations.
